packages/core/database/lib/fields/json.js

```diff
@@ -4,11 +4,25 @@ const Field = require('./field');
 
 class JSONField extends Field {
   toDB(value) {
+    if (typeof value !== 'object') {
```

Comment:
Hey, let's not forget that `2`, `'Hello'` or `true` should be valid JSON values too.

```js
const json = (value) => {
    const str = JSON.stringify(value);
    const prs = JSON.parse(str);
    console.log(value, str, prs)
}

json(2)
// 2 '2' 2

json('Hello')
// Hello "Hello" Hello

json(true)
// true 'true' true

json(null)
// null 'null' null

json({ foo: 'bar' })
// { foo: 'bar' } '{"foo":"bar"}' { foo: 'bar' }
```

If there is an issue while parsing what we get from the DB, this is another problem.

Reply:
although formally correct in a way, this is what originally lead to this error. 
The intent of Strapi's `JSON` datatype seems to be storing _objects_, for `2`, `'Hello'` or `true` there are more suitable datatypes around I think.

Reply:
If we want to allow ints, strings and booleans why are we even parsing the JSON at all? Shouldn't we just return the string value from the db?